e2e/src/scenarios/download.e2e-spec.ts

```diff
@@ -0,0 +1,49 @@
+import { AppPage } from '../app.po';
+
+describe('download', () => {
+  let page: AppPage;
+
+  beforeEach(() => {
+    page = new AppPage();
+  });
+
+
+  it('should download the pdf', () => {
+    const file = 'e2e/src/downloads/example.pdf';
+
+    page.getPdfViewer().click();
+
+    const downloadPage = page.getDownloadButton();
+    downloadPage.click()
+      .then(() => page.wait(3000))
```

Comment:
should we use await rather than .then?

Reply:
youre right, then again, i dont think the then should be there as page.wait would do the same thing

Reply:
Well the .then comment applies to the whole PR really. I don't think we should be using them when we have `await`. 

The page.wait function will probably still be needed as that's to wait for a specific amount of time (I assume to let the HTML update).

Reply:
yh for html update and in this case its to wait for the file to be downloaded